sched/pthread/pthread_condwait.c

```diff
@@ -107,7 +107,7 @@ int pthread_cond_wait(FAR pthread_cond_t *cond, FAR pthread_mutex_t *mutex)
        * or if the thread is canceled (ECANCELED)
        */
 
-      status = pthread_sem_take((FAR sem_t *)&cond->sem, NULL, false);
+      status = nxsem_wait_uninterruptible(&cond->sem);
```

Comment:
I agree with all except this change. `pthread_sem_take` returns `-` status of `nxsem_wait_uninterruptible`, so error handling of `pthread_cond_wait` will be changed.

Reply:
Yes, you are right. let me drop the change from this file.